docs/api-reference.md

```
@@ -621,16 +621,16 @@ Decrypt AES-256-CBC encrypted data.
 ### Key Wrapping
 
 ```ts
-getEncryptionData(): EncryptionData
+getEncryptionData(): Promise<EncryptionData>
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)